src/Compiler/RefC/RefC.idr

```diff
@@ -887,7 +887,7 @@ createCFunctions n (MkACon tag arity nt) = do
 
 
 createCFunctions n (MkAForeign ccs fargs ret) = do
-  case parseCC ["C"] ccs of
+  case parseCC ["RefC", "C"] ccs of
```

Comment:
Given that `RefC` will come first but `C` will act as a fallback, is it really necessary
to make all the foreign bindings in `System.Clock`, `String.Iterator`, etc. `RefC`-only?

Reply:
Given that they can't actually be used by other backends, yes.

That said, with some rearrangement, they could be made to work for other backends. I based my tagging on the current state of the code.